common/tools/mocha-multi-reporter.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+"use strict";
+
+const Mocha = require("mocha");
+const MochaJUnitReporter = require("mocha-junit-reporter");
```

Comment:
One thing to note here is that the `mocha-multi-reporter.js` file is not part of any project/utilities, it is an independent file and doesn't have a `package.json` of its own. 
Placed here(under `common/tools/` folder) only so that all the SDKs can leverage.

SDKs are expected to import `mocha` and `mocha-junit-reporter` as devDependencies and then use it as part of the mocha test command in `package.json`.